.jenkins

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent {
+        label 'DoNotRunAnythingOnMaster'
+    }
+    stages {
+        stage("Documentation") {
+            agent {
+                docker {
+                    image "ubuntu:18.04"
+                    label 'docker'
+                }
+            }
+            environment {
+                GIT_CREDS = credentials('0f170146-bd5e-44c0-a6e0-4f6fac3230fa')
+            }
+            steps {
+                sh 'apt update && apt install --no-install-recommends -y build-essential cmake doxygen git'
+                sh 'git config http.sslVerify false'
+                sh '''rm -rf build_doc && mkdir -p build_doc && cd build_doc &&
+                      cmake -D Tasmanian_ENABLE_DOXYGEN=ON .. && make Tasmanian_doxygen &&
```

Comment:
Minor point, here we should add `-D DOXYGEN_INTERNAL_DOCS=ON`
```
cmake -D DOXYGEN_INTERNAL_DOCS=ON -D Tasmanian_ENABLE_DOXYGEN=ON; ...
```

Reply:
Done